app/Keycloak/Client/KeycloakApiClient.php

```diff
@@ -63,20 +61,21 @@ public function createClient(Realm $realm, Integration $integration, UuidInterfa
             throw KeyCloakApiFailed::failedToCreateClientWithResponse($response);
         }
 
-        $this->logger->info(sprintf('Client %s, client id %s created with id %s', $integration->name, $integration->id->toString(), $id->toString()));
+        $this->logger->info(sprintf('Client %s for realm %s created with id %s', $integration->name, $realm->publicName, $id->toString()));
     }
 
     /**
      * @throws KeyCloakApiFailed
      */
-    public function addScopeToClient(Realm $realm, UuidInterface $clientId, UuidInterface $scopeId): void
+    public function addScopeToClient(Client $client, UuidInterface $scopeId): void
     {
         try {
             $response = $this->client->sendWithBearer(
                 new Request(
                     'PUT',
-                    sprintf('admin/realms/%s/clients/%s/default-client-scopes/%s', $realm->internalName, $clientId->toString(), $scopeId->toString())
-                )
+                    sprintf('admin/realms/%s/clients/%s/default-client-scopes/%s', $client->getRealm()->internalName, $client->id->toString(), $scopeId->toString())
+                ),
+                $client->getRealm()
             );
         } catch (GuzzleException $e) {
```

Comment:
A special reason why this catches a `GuzzleException` and not a `Throwable` like all other methods?